docs/lib/content/configuring-npm/package-lock-json.md

```diff
@@ -114,10 +114,10 @@ the npm registry.  Lockfiles generated by npm v7 will contain
 * `1`: The lockfile version used by npm v5 and v6.
 * `2`: The lockfile version used by npm v7, which is backwards compatible
   to v1 lockfiles.
-* `3`: The lockfile version used by npm v7, _without_ backwards
-  compatibility affordances.  This is used for the hidden lockfile at
-  `node_modules/.package-lock.json`, and will likely be used in a future
-  version of npm, once support for npm v6 is no longer relevant.
+* `3`: The lockfile version supported by npm v7, _without_ backwards
```

Comment:
This is the version "used" by v9.

Reply:
Just to clarify, would you like me to simplify this part of the text?

1. I can remove the part about `node_modules/.package-lock.json` since that's only relevant to NPM internals if I understand correctly.
2. I can remove the distinction between backwards-compatible but mandatory use (as is the case for npm v7 and v8 regarding lockfile v2) and the "only for a new lockfile" use (as is the case for npm v9 regarding lockfile v3).

Reply:
Yes this should be simplified.

Reply:
Done.